.github/codecov.yml

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Test using:
+#   curl --data-binary @.github/codecov.yml https://codecov.io/validate
+#
+# https://docs.codecov.io/docs/codecovyml-reference
+codecov:
+  require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+  # Add status message to PRs.
+  status:
+    project:
+      python:
+        target: auto
+        threshold: 5%
+        base: auto
+        paths:
+          - "sdks/python"
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+# Do not comment on PRs.
```

Comment:
Why not?

Reply:
It used used to be spammy (1 message per push), but now it seems to be editing the original message so not that bad.
I am reconsidering.

Reply:
Oh okay, nice